src/components/Toasts/ToastsProvider.jsx

```diff
@@ -26,11 +26,12 @@ const ToastsProvider = ({ children }) => {
   };
 
   return (
+    // eslint-disable-next-line react/jsx-no-constructed-context-values
```

Comment:
Should this context value be computed with a `useMemo` to avoid ignoring this ESLint rule?